src/gpu/nvidia/sycl_cuda_utils.cpp

```diff
@@ -58,15 +58,35 @@ bool attr_post_ops_ok(const primitive_attr_t *attr) {
     }
 }
 
-bool has_bf16_support(const ::sycl::device &dev) {
-    // This function checks compute capabilities of the given device.
-    // BF16 is supported starting with compute capabilities 8.0.
+cudaDeviceProp query_device_properties(const ::sycl::device &dev) {
     auto cuda_dev = compat::get_native<CUdevice>(dev);
     cudaDeviceProp prop {};
     cudaGetDeviceProperties(&prop, cuda_dev);
+    return prop;
+}
```

Comment:
(random spot): please update the copyright banner to 2024 for this file.

Reply:
Copyright has been fixed.